### DISCUSSION TITLE
WSA Sideloader v1.4.3

### DISCUSSION THREAD

**Discussion by infinitepower18** [infinitepower18/WSA-Sideloader#96] [Announcements]
Thanks to the community, WSA Sideloader is now available in 3 additional languages:
- French by @ThomasGitHubcom 
- Arabic (Egypt) by @MesterPerfect 
- Brazilian Portuguese by @auryan 

<hr /><em>This discussion was created from the release <a href='https://github.com/infinitepower18/WSA-Sideloader/releases/tag/v1.4.3'>WSA Sideloader v1.4.3</a>.</em>

**Comment by MesterPerfect** [comment]
very good
